### ISSUE TITLE
project includes files that are missing

### ISSUE THREAD

**Issue by storkinsj** [rasa/regdelta#1]
I believe that tstl.h needs to be pulled in from another one of your repositories. If there is an easy way to do it (or not) it would be good to include that in the readme area.

**Comment by rasa** [comment]
Good point. Several of my C/C++ repos depend on https://github.com/rasa/shared
